### PR TITLE
Added hass-unraid

### PR DESCRIPTION
**Pull request by PilaScat**

### CODE REVIEW

File: templates/hass-unraid [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <Support>https://github.com/IDmedia/hass-unraid/issues</Support>
+  <Project>https://github.com/IDmedia/hass-unraid</Project>
+  <Overview>This Docker container parses and forwards all WebSocket messages from your Unraid server to Home Assistant using the MQTT protocol. This enables you to create dashboards that provide a superior overview compared to Unraid's native capabilities.&#xD;</Overview>
+  <Beta>True</Beta>
+  <Category>HomeAutomation: Tools:</Category>
+  <Icon>https://raw.githubusercontent.com/IDmedia/hass-unraid/main/extras/logo.png</Icon>
+  <ExtraParams>--restart always</ExtraParams>
+  <Requires>NOT WORKING WITH SELF SIGNED CERTIFICATES&#xD;Ensure that Home Assistant and MQTT are correctly configured before proceeding.&#xD;For configuration please refer to the GitHub page.</Requires>
+  <Config Name="Path" Target="/data" Default="/mnt/user/appdata/hass-unraid" Mode="rw" Description="" Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata/hass-unraid</Config>
+  <Config Name="Timezone" Target="TZ" Default="Europe/Oslo" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">Europe/Oslo</Config>
```
Comment (nwithan8) [inline templates/hass-unraid:17]: Remove TZ, Unraid passes this into all Docker containers automatically

File: templates/hass-unraid [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <Privileged>false</Privileged>
+  <Support>https://github.com/IDmedia/hass-unraid/issues</Support>
+  <Project>https://github.com/IDmedia/hass-unraid</Project>
+  <Overview>This Docker container parses and forwards all WebSocket messages from your Unraid server to Home Assistant using the MQTT protocol. This enables you to create dashboards that provide a superior overview compared to Unraid's native capabilities.&#xD;</Overview>
+  <Beta>True</Beta>
+  <Category>HomeAutomation: Tools:</Category>
+  <Icon>https://raw.githubusercontent.com/IDmedia/hass-unraid/main/extras/logo.png</Icon>
+  <ExtraParams>--restart always</ExtraParams>
+  <Requires>NOT WORKING WITH SELF SIGNED CERTIFICATES&#xD;Ensure that Home Assistant and MQTT are correctly configured before proceeding.&#xD;For configuration please refer to the GitHub page.</Requires>
+  <Config Name="Path" Target="/data" Default="/mnt/user/appdata/hass-unraid" Mode="rw" Description="" Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata/hass-unraid</Config>
```
Comment (nwithan8) [inline templates/hass-unraid:16]:
```suggestion
  <Config Name="Path" Target="/data" Default="/mnt/user/appdata/hass-unraid" Mode="rw" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/mnt/user/appdata/hass-unraid</Config>
```

Appdata config paths should be under Advanced as users are not expected to ever change this.

File: templates/hass-unraid [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  <Registry>ghcr.io/idmedia/hass-unraid</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/IDmedia/hass-unraid/issues</Support>
+  <Project>https://github.com/IDmedia/hass-unraid</Project>
+  <Overview>This Docker container parses and forwards all WebSocket messages from your Unraid server to Home Assistant using the MQTT protocol. This enables you to create dashboards that provide a superior overview compared to Unraid's native capabilities.&#xD;</Overview>
+  <Beta>True</Beta>
+  <Category>HomeAutomation: Tools:</Category>
+  <Icon>https://raw.githubusercontent.com/IDmedia/hass-unraid/main/extras/logo.png</Icon>
+  <ExtraParams>--restart always</ExtraParams>
```
Comment (nwithan8) [inline templates/hass-unraid:14]: Remove these parameters. Restart policies should be handled via the Docker system in Unraid, not on a per-container basis.

**Review by nwithan8 — CHANGES_REQUESTED** [review]

**Review by nwithan8 — APPROVED** [review]

**Review by Roxedus — APPROVED** [review]

**Comment by nwithan8** [comment]
> do I need to do anything?

The GitHub Action here is stuck unfortunately. I tried to forcibly retrigger it with an empty commit, but no dice.

If you open a new PR with the same code, I'll stamp it and get it merged in, hopefully won't run into the same issue.

**Comment by PilaScat** [comment]
I don't know why but there isn't in the apps tab

**Comment by nwithan8** [comment]
This probably explains why the Action had stalled, the file is missing the `.xml` suffix, so it didn't get picked up in the feed: https://github.com/Squidly271/AppFeed/blob/master/log.txt#L1521

**Comment by PilaScat** [comment]
I didn't notice, thanks

**Pull request merged**